templates/docs/base/typography.md

```diff
@@ -152,6 +152,23 @@ View example of the base blockquotes
 View example of the small text
 </a></div>
 
+### Extra small capitalized text
+
+Extra small capitalized text is used to style column headers in tables. This syling has proved useful in other contexts. To apply it, add the class `p-text--x-small-capitalized`.
```

Comment:
Should be "capitalised", english spelling

Reply:
fixed